scoper.inc.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * PHP-Scoper configuration file.
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+use Isolated\Symfony\Component\Finder\Finder;
+
+// Google API services to include classes for.
+$google_services = implode(
+	'|',
+	array_map(
+		function( $service ) {
+			return preg_quote( $service, '#' );
+		},
+		array(
+			'Analytics',
+			'AnalyticsReporting',
+			'AdSense',
+			'Pagespeedonline',
+			'PeopleService',
+			'SiteVerification',
+			'TagManager',
+			'Webmasters',
+		)
+	)
```

Comment:
This `array_map` seems unnecessary. Everything within is already regex-safe.

Reply:
I mostly put it there to be safe for future changes, but agreed, we can also do so once actually needed.